vita3k/gui/src/game_install_dialog.cpp

```diff
@@ -70,14 +70,14 @@ void draw_game_install_dialog(GuiState &gui, HostState &host) {
         ImGui::Separator();
         ImGui::Spacing();
         ImGui::SetCursorPosX(ImGui::GetWindowWidth() / 2 - 65);
-        if (ImGui::Button("Yes", BUTTON_SIZE)) {
+        if (ImGui::Button("Yes", BUTTON_SIZE))
```

Comment:
Why did you remove this? It makes more sense with it... Using just indentation to convey stuff can make things confusing. For multiline ifs, use braces please.

Reply:
What ? Have only one value return on if so is not needed i have juste forget remove it after test, look other part on code already have this removed is just normal